test/lib.list.js

```diff
@@ -89,8 +89,7 @@ describe('List method', () => {
       assert.isString(app.androidVersionText);
       assert.isString(app.androidVersion);
       assert.isString(app.contentRating);
-
-      assert.equal(app.priceText, 'Free');
+      assert.equal(app.priceText, '');
```

Comment:
Again, this is failing because of a bug, if you don't fix the bug then don't hide the error.

Reply:
On the HTTP response, I get the following 

[Resp.txt](https://github.com/facundoolano/google-play-scraper/files/1988037/Resp.txt)

Its empty and not "Free" -- [0,"USD",""].

Reply:
Yes it's empty, but the code was supposed to be setting it to free. It's a bug we need to fix, so the test should keep failing until we do.

Reply:
https://github.com/facundoolano/google-play-scraper/issues/230